nimsuggest/nimsuggest.nim

```diff
@@ -736,12 +736,8 @@ else:
     var retval: seq[Suggest] = @[]
     let conf = nimsuggest.graph.config
     conf.ideCmd = cmd
-    conf.writelnHook = proc (line: string) =
-      retval.add(Suggest(section: ideMsg, doc: line))
     conf.suggestionResultHook = proc (s: Suggest) =
       retval.add(s)
-    conf.writelnHook = proc (s: string) =
-      stderr.write s & "\n"
```

Comment:
But this changes the behavior, doesn't it?

Reply:
yeah, maybe  let initNimsuggest take a conf arg to resolve this

Reply:
the bad part is current implementation will print duplicated messages to stderr that cant be stoped.